src/routes/sendEmail.routes.js

```diff
@@ -0,0 +1,9 @@
+import { Router } from 'express';
+import { sendEmail } from '../controller/requests/sendEmailReq.js';
+
+const sendEmailRoute = Router();
+
+// Define a route for sending emails
```

Comment:
I suggest that it is not necessary to comment on the logic in this route because the entire route package follows the same logic.

Reply:
Done